jobs/roleNotifier.js

```diff
@@ -0,0 +1,23 @@
+import { Role } from "../models";
+import { roleNotifierMail } from "./templates";
+
+export default async transporter => {
+
+    if(!transporter)
+        return;
+
+    const inactiveRoles = await Role.find({newlyAdded: true});
```

Comment:
Better will be checking date of add new role and compare it with current date. I think new property is redundant

Reply:
That's true, but we would have to store the date of when we added the role, and since we are deleting the field as soon as we don't need it, it's not that big of a deal

Reply:
I'm sure that every model in mongoose have native fields like date of creating or updating. Could you check this?

Reply:
https://stackoverflow.com/a/15147350/10908022
we can turn that on, but that's not the default. Do you think we should do that?

Reply:
Yes, I think we should activate these properties for every models